### ISSUE TITLE
UP019 should detect `typing_extensions.Text`

### ISSUE THREAD

**Issue by dscorbett** [astral-sh/ruff#20819]
### Summary

[`typing-text-str-alias` (UP019)](https://docs.astral.sh/ruff/rules/typing-text-str-alias/) detects `typing.Text`. It would also be useful for it to detect `typing_extensions.Text`. [Example](https://play.ruff.rs/c9b704ed-1e1e-45aa-8e5a-de32fd3280ac):
```console
$ cat >up019.py <<'# EOF'
from typing_extensions import Text
foo: Text = "bar"
# EOF

$ ruff --isolated check up019.py --select UP019
All checks passed!
```

### Version

ruff 0.14.0 (beea8cdfe 2025-10-07)
